modules/lambdas/variables.tf

```diff
@@ -89,14 +89,24 @@ variable "orca_sqs_staged_recovery_queue_arn" {
   description = "The ARN of the staged-recovery-queue SQS"
 }
 
-variable "orca_sqs_status_update_queue_arn" {
+variable "orca_sqs_archive_recovery_queue_arn" {
   type        = string
-  description = "The ARN of the status-update-queue SQS"
+  description = "The ARN of the archive-recovery-queue SQS"
 }
 
 variable "orca_sqs_status_update_queue_id" {
   type        = string
-  description = "The URL of the SQS queue that recoery status updates are read from/posted to."
+  description = "The URL of the SQS queue that recovery status updates are read from/posted to."
+}
+
+variable "orca_sqs_archive_recovery_queue_id" {
+  type        = string
+  description = "The SQS URL that triggers post_copy_request_to_queue upon successful object restore from archive bucket. Also receives input from request_from_archive for files already recovered."
+}
+
+variable "orca_sqs_status_update_queue_arn" {
```

Comment:
Reorder from

archive arn
status id
archive id
status arn

to

archive arn
archive id
status arn
status id

Reply:
Looking closer, the archive entries should also go further up on the list due to alphabetization.

Reply:
fixed

Reply:
arn should come before id

Reply:
oops fixed